packages/graphql-toolbox/src/contexts/auth.tsx

```diff
@@ -79,13 +80,16 @@ export function AuthProvider(props: any) {
             loginPayload = loginPayloadFromDesktop;
             setValue((v) => ({ ...v, isNeo4jDesktop: true }));
         } else {
-            const storedEncryptedPayload = Storage.retrieve(LOCAL_STATE_LOGIN);
-            if (storedEncryptedPayload && typeof storedEncryptedPayload === "string") {
-                const { encryptedPayload, hashKey } = JSON.parse(storedEncryptedPayload as string);
-                loginPayload = decrypt(encryptedPayload, hashKey) as unknown as LoginPayload;
+            const storedConnectionUsername = Storage.retrieve(LOCAL_STATE_CONNECTION_USERNAME);
+            const storedConnectionUrl = Storage.retrieve(LOCAL_STATE_CONNECTION_URL);
+            if (storedConnectionUrl && storedConnectionUsername) {
+                loginPayload = {
+                    username: storedConnectionUsername,
+                    url: storedConnectionUrl,
+                };
             }
         }
-        if (loginPayload && value && !value.driver) {
+        if (loginPayload && loginPayload.password && value && !value.driver) {
```

Comment:
Can you use optional chaining here?

```suggestion
        if (loginPayload?.password && value && !value.driver) {
```